src/main/scala/tech/sourced/gemini/FeaturesHash.scala

```diff
@@ -0,0 +1,62 @@
+package tech.sourced.gemini
+
+import tech.sourced.featurext.generated.service.Feature
+
+import scala.collection.mutable
+
+object FeaturesHash {
+  // All params below must be the same for hash & query
+  val defaultSeed = 0
```

Comment:
Constants usually use `CamelCase` if using [Scala-style](https://docs.scala-lang.org/style/naming-conventions.html) or `UPPER_CASE` if using Java-style (which some people uses in Scala too).

Reply:
@bzz should we change it everywhere?

Reply:
If you end up deciding to change it everywhere, it's probably better to do it on its own PR.

Reply:
If nobody has a strong opinion about this, I belive it is good as-is and there is no need to change anything as it follows the styleguide:

> That is, each word is capitalized, except possibly the first word